src/Generator/Example/GeneratorExample/Interfaces/IClassA.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GeneratorExample.Interfaces;
+
+public interface IClassA { }
```

Comment:
I'm assuming this change in test is related to the issue that you're trying to fix. Do you mind adding some PR description that explains what's going on?

Reply:
Sure i will add the description in the PR description :)